src/realm/object-store/impl/collection_change_builder.cpp

```diff
@@ -26,12 +26,14 @@ using namespace realm;
 using namespace realm::_impl;
 
 CollectionChangeBuilder::CollectionChangeBuilder(IndexSet deletions, IndexSet insertions, IndexSet modifications,
-                                                 std::vector<Move> moves, bool root_was_deleted)
+                                                 std::vector<Move> moves, bool collection_was_cleared,
```

Comment:
Adding the new argument in this position means that any code which was calling this constructor and passing in a bool will still compile, but now do something different.